### DISCUSSION TITLE
Webots Force Feedback

### DISCUSSION THREAD

**Discussion by archiebaxter** [cyberbotics/webots#6039] [General]
I am trying to monitor how a robotic arm with a gripper reacts to different payload weights, geometries, and orientations. I am confident that I have created my models correctly in terms of geometry and simulation. However, the value I get for 'ForceFeedback' on the coupled linear actuator motor I am using for my gripper fingers does not change when I vary the mass of the object it is carrying. This seems incorrect to me because surely as mass increases gripping force should too. 
-  For example, when I increase the mass of the 'redbox', which I am picking up with the robotic gripper, from 10kg to 100kg the force 
   feedback value from my gripper motor stays at 2989N with little variation during the simulation. 

Does anyone know why this is ? 
Is it to do with the value of MaxForce set in the motors ?

[Two_Finger_Gripper.zip](https://github.com/cyberbotics/webots/files/11120138/Two_Finger_Gripper.zip)
I have attached my Webots file, 'Two Finger Gripper New' is my world and 'two_finger_gripper_controller' is my controller.

**Comment by archiebaxter** [comment]
@ad-daniel @omichel @DrakerDG

**Comment by omichel** [comment]
Duplicate with #6018.
